lib/evaluators/AsyncEvaluator.ts

```diff
@@ -23,8 +23,15 @@ export class AsyncEvaluator {
   private readonly evaluator: IExpressionEvaluator<E.Expression, Promise<E.TermExpression>>;
 
   public static completeContext(context: IAsyncEvaluatorContext): ICompleteAsyncEvaluatorContext {
+    const now = context.now || new Date(Date.now());
+    let defaultTimeZone = context.defaultTimeZone;
+    if (!defaultTimeZone) {
+      // We make use of wrong modulo implementation: (-90) % 60 = -30
```

Comment:
Why is it wrong?

Reply:
Yes, so it's not really wrong since most programming languages work like this but the mathematical modulo operator would return a positive integer. 

Would you want me to remove this comment?

Reply:
Perhaps yes, to avoid confusion